src/toil/test/utils/toilKillTest.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2018 Regents of the University of California
```

Comment:
@kannon92 Please change this to you or your copyright holder

Reply:
Oh.  That was silly of me.  I copied this from an older example and didn't update the header.  I think the update should suffice.